### PR TITLE
8267075: jcmd VM.cds should print directory of the output files

### PR DESCRIPTION
**Pull request by yminqi**
Hi, Please review the simple change for jcmd VM.cds to print directory of output file.
The printout will print out absolute file path, also a test scenario added.

Tests: tier1,tier2,tier3,tier4

Thanks
Yumin

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8267075](https://bugs.openjdk.java.net/browse/JDK-8267075): jcmd VM.cds should print directory of the output files

### Reviewers
 * [Calvin Cheung](https://openjdk.java.net/census#ccheung) (@calvinccheung - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/4576/head:pull/4576` \
`$ git checkout pull/4576`

Update a local copy of the PR: \
`$ git checkout pull/4576` \
`$ git pull https://git.openjdk.java.net/jdk pull/4576/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 4576`

View PR using the GUI difftool: \
`$ git pr show -t 4576`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/4576.diff">https://git.openjdk.java.net/jdk/pull/4576.diff</a>

</details>

### CODE REVIEW

File: test/hotspot/jtreg/runtime/cds/appcds/jcmd/JCmdTestFileSafety.java
Comment (calvinccheung) [inline test/hotspot/jtreg/runtime/cds/appcds/jcmd/JCmdTestFileSafety.java:48]: Could you change the variable names to `promptStdout` and `promptStderr` instead?

Please also add the bug id to the `@bug`.

File: test/hotspot/jtreg/runtime/cds/appcds/jcmd/JCmdTestFileSafety.java
Comment (calvinccheung) [inline test/hotspot/jtreg/runtime/cds/appcds/jcmd/JCmdTestFileSafety.java:53]: What if either `promtStdout` or `promtStderr` is not found in stdout, should a RuntimeException be thrown as well?

**Review by calvinccheung — COMMENTED** [review]
Code changes look good. Just couple of comments in the test case.

File: test/hotspot/jtreg/runtime/cds/appcds/jcmd/JCmdTestFileSafety.java
Comment (yminqi) [reply test/hotspot/jtreg/runtime/cds/appcds/jcmd/JCmdTestFileSafety.java:53]: The exception could be other type and without containing those two strings. It should leave to the default processing. Only with those two messaged found, we check if the file with a absolute path.

**Review by calvinccheung — APPROVED** [review]
Looks good.

**Comment by yminqi** [comment]
@calvinccheung Thanks for review! Since it is a simple fix, I will integrate without second review.
/integrate

**Pull request closed without merging**
